src/main/java/com/google/alpollo/AnalysisHelper.java

```diff
@@ -10,6 +10,7 @@
 import com.google.cloud.language.v1.Document;
 import com.google.cloud.language.v1.EncodingType;
 import com.google.cloud.language.v1.Entity;
+import com.google.cloud.language.v1.EntityMention;
```

Comment:
nit: You added a new import, but doesn't seem to be using it... Clean the unused imports